### PR TITLE
Update topology notification message

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2276438

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
image

File: packages/odf/components/topology/Topology.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ const MessageButton: React.FC = () => {
     <div className="odf-topology__message-button">
       <BlueInfoCircleIcon />{' '}
       {showMessage &&
-        t('This view is only supported for Internal mode cluster.')}{' '}
+        t('This view is not available for external mode cluster.')}{' '}
```
Comment (alfonsomthd) [inline packages/odf/components/topology/Topology.tsx:128]: @TimothyAsirJeyasing Should be `external` capitalized like `Internal was before` ?

Comment (TimothyAsirJeyasing) [reply packages/odf/components/topology/Topology.tsx:128]: This was given by the Doc team, Sure i will check with them once.

Comment (TimothyAsirJeyasing) [reply packages/odf/components/topology/Topology.tsx:128]: @alfonsomthd I've checked with our documentation team, and they confirmed that 'external' should be in lowercase, following our current style guidelines.

**Pull request merged**